Flight simulator sample - Physics update (#19)
Esri/arcgis-maps-sdk-unreal-engine-samples: sample_project/Content/SampleViewer/Samples/FlightSimulator/readme.md

4. Create a flying object as a **Pawn** and attach the [**ArcGIS Camera Component**](https://developers.arcgis.com/unreal-engine/maps/camera/#arcgis-camera-component) and [**ArcGIS Location Component**](https://developers.arcgis.com/unreal-engine/maps/location-component/).
5. Connect the flying object to the [**Player Controller**](https://docs.unrealengine.com/5.0/en-US/player-controllers-in-unreal-engine/) and place it on the map. In this sample, the flying object is saved as **BP_Ship**.
6. Work on [physics](https://docs.unrealengine.com/5.0/en-US/physics-in-unreal-engine/) from Unreal Engine's feature for the flying object. This sample uses [**Set Physics Linear Velocity**](https://docs.unrealengine.com/5.0/en-US/BlueprintAPI/Physics/SetPhysicsLinearVelocity/) for the speed control and [**Add Torque in Degrees**](https://docs.unrealengine.com/4.26/en-US/BlueprintAPI/Physics/AddTorqueinDegrees/) for the object's roll, pitch, and yaw. If the plane reduces enough its speed, false gravity drags the plane toward Earth.

## Controls
This sample supports keyboard controls and use of an Xbox/Playstation controller

> Keyboard Controls: Spacebar for Acceleration, Q for Yaw Left, E for Yaw Right, W or Mouse Y for Pitch Up, S or Mouse Y for Pitch Down, A or Mouse X for Roll Left, D or Mouse X for Roll Right. 
> H can be used to toggle the landing gear.

Author: ```suggestion
```

> Controller Controls: Right Trigger for Acceleration, Left Trigger for Deceleration, Left Stick Left and Right for Yaw, Right Stick Left and Right for Roll, Right Stick up and down for Pitch. Right Thumbstick button to toggle landing gear. 

Author: ```suggestion
```

## How it works

1. Check the box for **Mesh Colliers Enabled** on the `ArcGIS Map` component.

> 2. Make sure the component is in **Global Mode**.
> 3. Pick an airport to take off from and set the latitude and longitude of the airport as your world center
> 4. Create a new blueprint and call it "bp_Runway".
> 5. Add a cube and change the scale of it to be 50, 2500, 1. Or something similar that suits your needs.
> 6. Add your Plane blueprint on top of your run way.
> 7. Enjoy flying around the world.
> 
> Movement of the plane is done through Unreal physics.
> We add a physics **linear velocity** depending on the thrust and rotate for each of the axis adding a **torque** in such axis so that the plane can rotate in the roll, pitch and yaw axes. If the plane reduces enough the speed, a false gravity drags the plane towards Earth.

Reviewer: ```suggestion
2. Set the **Origin Position** for the place where you want to place the runway.
3. Create a runway and add it to the level. In this sample, the runway is created by using a cube and saved as **BP_Runway**.
4. Create a flying object as a **Pawn** and attach the [**ArcGIS Camera Component**](https://developers.arcgis.com/unreal-engine/maps/camera/#arcgis-camera-component) and [**ArcGIS Location Component**](https://developers.arcgis.com/unreal-engine/maps/location-component/).
5. Connect the flying object to the [**Player Controller**](https://docs.unrealengine.com/5.0/en-US/player-controllers-in-unreal-engine/) and place it on the map. In this sample, the flying object is saved as **BP_Ship**.
6. Work on [physics](https://docs.unrealengine.com/5.0/en-US/physics-in-unreal-engine/) from Unreal Engine's feature for the flying object. This sample uses [**Set Physics Linear Velocity**](https://docs.unrealengine.com/5.0/en-US/BlueprintAPI/Physics/SetPhysicsLinearVelocity/) for the speed control and [**Add Torque in Degrees**](https://docs.unrealengine.com/4.26/en-US/BlueprintAPI/Physics/AddTorqueinDegrees/) for the object's roll, pitch, and yaw. If the plane reduces enough its speed, false gravity drags the plane toward Earth.
```

This is how it looks like to me though I might be wrong...

## Controls
This sample supports keyboard controls and use of an Xbox/Playstation controller
Keyboard Controls: Spacebar for Acceleration, Q for Yaw Left, E for Yaw Right, W or Mouse Y for Pitch Up, S or Mouse Y for Pitch Down, A or Mouse X for Roll Left, D or Mouse X for Roll Right. 
H can be used to toggle the landing gear.

> Controller Controls: Right Trigger for Acceleration, Left Trigger for Deceleration, Left Stick Left and Right for Yaw, Right Stick Left and Right for Roll, Right Stick up and down for Pitch. Right Thumbstick button to toggle landing gear. 

Reviewer: I added the control to the How to use the sample section above.

## Controls
This sample supports keyboard controls and use of an Xbox/Playstation controller

> Keyboard Controls: Left Shift for Acceleration, Q for Yaw Left, E for Yaw Right, W or Mouse Y for pitch Up, S or Mouse Y for pitch down, A or Mouse X for roll left, D or Mouse X for roll right. 

Reviewer 2: ```suggestion
Keyboard Controls: Spacebar for Acceleration, Q for Yaw Left, E for Yaw Right, W or Mouse Y for Pitch Up, S or Mouse Y for Pitch Down, A or Mouse X for Roll Left, D or Mouse X for Roll Right. 
```

> H can be used to toggle the landing gear.

Reviewer 2: This doesn't work for me. Or I don't see anything happening.
